dishdive/components/map.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+import { GoogleMap, Marker, LoadScript } from '@react-google-maps/api';
+
+const MapComponent = ({ location }) => {
```

Comment:
usually component name and file names are same
```suggestion
const Map = ({ location }) => {
```